addons/io_hubs_addon/preferences.py

```diff
@@ -165,6 +165,101 @@ def execute(self, context):
         return {'FINISHED'}
 
 
+def reload_user_components():
+    from .components.components_registry import unload_user_components, load_user_components
+    unload_user_components()
+    load_user_components()
+
+
+class HubsUserComponentsPath(bpy.types.PropertyGroup):
+    name: StringProperty(
+        name='User components path entry name',
+        description='The user components path entry name.',
+    )
+    path: StringProperty(
+        name='User components path path',
+        description='The user components path path. You can copy external components here.',
```

Comment:
I think this description could be improved.  How about something like:
```suggestion
        description='The path to a user defined component definitions directory.  You can copy external components here and they will be loaded automatically',
```

Reply:
Added